docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -142,8 +146,8 @@ The following table lists the streaming protocol features supported by the playe
 | Streaming protocol                  | Supported feature |
 |-------------------------------------|-------------------|
 | Hypertext Transfer Protocol (HTTP)  | HTTP Streaming    |
-|  | HTTP Live Streaming                 |                   |
-|  | HTTP Progressive Download Play      |                   |
+| HTTP Live Streaming                 |                   |
```

Comment:
line 149: Fill in the right value. ('HTTP Live Streaming' to 'Supported feature' column)
line 150: Fill in the right value. ('HTTP Progressive Download Play' to 'Supported feature' column )

Reply:
The same word "supported features" duplicated three times: a sentence, caption of table, and table title.
I changed table to lists.